### ISSUE TITLE
Editing style via DevTools breaks all styles

### ISSUE THREAD

**Issue by zakkor** [vercel/styled-jsx#668]
#### Do you want to request a *feature* or report a *bug*?
Report a bug, posting another issue because #513 was closed and is not getting attention anymore

#### What is the current behavior?
Changing styles in Chrome devtools disables all the styles, and in Firefox nothing happens

#### If the current behavior is a bug, please provide the steps to reproduce and possibly a minimal demo or testcase in the form of a Next.js app, CodeSandbox URL or similar

Any app using styled-jsx

#### What is the expected behavior?
I should be able to change the styles in devtools

#### Environment (include versions)

* OS: Any
* Browser: Chrome, Firefox
* styled-jsx (version): 3.3.0

#### Did this work in previous versions?
No

**Comment by FelixTellmann** [comment]
I've experienced the same issue before. I figured that it only happens when having multiple '<style jsx>' elements in the same component.

**Comment by Divlo** [comment]
Same problem.

**Comment by Divlo** [comment]
I have the same problem, it's really annoying while developing...
But I have no idea how it occurs.

**Comment by jonluca** [comment]
This is still happening. 

I can repro with a component that only has `style jsx` element @FelixTellmann. I'm not sure if it's multiple nested versions of this? 

It breaks _all_ styles, too, not just the one you're editing, which is what makes it so frustrating

**Comment by dangelion** [comment]
Same here, really annoying. Any updates on this?

**Comment by darbymanning** [comment]
Yep, have the same issue. I was trying styled-jsx on a new project, but sadly this issue is too going to be too problematic to work around. Having the ability to toggle CSS properties in devtools is really essential.

**Comment by pixelize** [comment]
Same issue, any fixes around please ? :)

**Comment by siolfyr** [comment]
Could it be because of the source-maps?
